### DISCUSSION TITLE
Windows issue - Failed to encrypt: Access is denied (0x5)

### DISCUSSION THREAD

**Discussion by jmetterrothan** [cypress-io/cypress#19307] [Questions and Help]
Hello,
I'm facing an issue with the latest cypress 9 version (worked fine until 8.7.0) while running my tests on Windows. I did not find anything useful after looking on the internet for a solution.

Right after running the command `cypress run-ct` I get:

```
[7860:1208/113058.161:ERROR:os_crypt_win.cc(70)] Failed to encrypt: Access is denied. (0x5)
```

 I don't even understand what the problem could be.

**Comment by AltoRetrato** [comment]
I believe that after all those years you have already solved your issue one way or another, but if you still don't know the cause, I had the same error on many Chromium Embedded Framework (CEF) programs, including Google Chrome. The error was happening after calling the Windows function CryptProtectData(). In my case, the `%APPDATA%\Microsoft\Protect\CREDHIST` file was set as "read-only", and reverting that solved the issue.

For more details, see https://gist.github.com/AltoRetrato/05f6957756931b12dc7ff8a87c2fe835
